docs/tables/github_organization.md

```diff
@@ -16,6 +16,12 @@ The `github_organization` table provides insights into Organizations within GitH
 - You must specify the `login` column in `where` or `join` clause to query the table.
 - To list organizations that you are a member of, use the `github_my_organization` table.
 
+To query this table using a [fine-grained access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-fine-grained-personal-access-token), the following permissions are required (the token must be created under the resource owner organization):
+  - Organization permissions:
```

Comment:
Please add the missing permission

Reply:
Actually, the `Members` permission is required to populate all the column values. Updated the doc.